docs/src/man/getting_started.md

```diff
@@ -424,6 +407,49 @@ julia> df[(df.A .> 500) .& (300 .< df.C .< 400), :]
 │ 99  │ 797   │ 8     │ 399   │
 ```
 
+It is also possible to use a regular expression as a selector of columns matching it:
+```jldoctest dataframe
+julia> df = DataFrame(x1=1, x2=2, y=3)
+1×3 DataFrame
+│ Row │ x1    │ x2    │ y     │
+│     │ Int64 │ Int64 │ Int64 │
+├─────┼───────┼───────┼───────┤
+│ 1   │ 1     │ 2     │ 3     │
+
+julia> df[!, r"x"]
+1×2 DataFrame
+│ Row │ x1    │ x2    │
+│     │ Int64 │ Int64 │
+├─────┼───────┼───────┤
+│ 1   │ 1     │ 2     │
+```
+
+Finally, you can use `Not` and `All` selectors in more complex cases. The following examples move all columns containing `"x"` respectively to the front and to the end of a data frame:
+```
+julia> df = DataFrame(r="r", x1="x1", x2="x2", y="y")
```

Comment:
Maybe use number values instead of repeating column names?

Reply:
OK. I will fix this. Anyway, this PR should be updated after https://github.com/JuliaData/DataFrames.jl/pull/1971 is merged.

It is in general non blocking for 0.20 release anyway. Thank you!